### ISSUE TITLE
UC Browser is incorrectly identified as Chrome

### ISSUE THREAD

**Issue by pkdetlefsen** [oalders/http-browserdetect#172]
It appears like UC Browser is incorrectly identified as Chrome since the User Agent may contain `Chrome/` before `UCBrowser/`.

Example:

```
Mozilla/5.0 (Linux; U; Android 6.0.1; zh-CN; F5121 Build/34.0.A.1.247) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/40.0.2214.89 UCBrowser/11.5.1.944 Mobile Safari/537.36
```
I believe the solution is to check for `ucbrowser` before `chrome/` similar to `brave` and `silk`.

**Comment by oalders** [comment]
Thanks! I'd accept a pull request for this. :)

**Comment by konstantin-pustovalov-qs** [comment]
Hey! I am interested in the fix too. Please review my PR - https://github.com/oalders/http-browserdetect/pull/189 Thanks!

**Comment by oalders** [comment]
Closed via #189
